_data/link.yml

```diff
@@ -15,8 +15,9 @@ Transform:
 
 AggregateOp:
   link: "aggregate.html#ops"
-Bin:
-  link: bin.html
+BinParams:
+  name: Bin
```

Comment:
`Bin` is `boolean | BinParams` so this technically undoes what I intended to change.  Please remove this line.